pkgs/by-name/ag/age-plugin-openpgp-card/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+
+  rustPlatform,
+
+  pcsclite,
+  pkg-config,
+  ...
+}:
+rustPlatform.buildRustPackage (final: {
+  pname = "age-plugin-openpgp-card";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "wiktor-k";
+    repo = final.pname;
```

```suggestion
    repo = "age-plugin-openpgp-card";
```
This is an anti pattern as only this exact string will be true and very likely not any other

---

agreed